environments/staging.yaml

```diff
@@ -36,6 +36,31 @@ jobs:
         }
       }
     }
+  traits:
+    releaseGitHubPullRequestDiscovery: |-
```

Comment:
Is it OK to not have it in production.yaml? Aka in the non staging environment, `{{- .Values.jobs.traits.releaseGitHubPullRequestDiscovery | nindent 12 }}` won't pollute the resulting YAML

Reply:
Yes it is indeed an issue.